google/cloud/bigquery/v2/minimal/internal/common_v2_resources.cc

```diff
@@ -634,11 +634,11 @@ void to_json(nlohmann::json& j, ColumnData const& c) {
   j = nlohmann::json{{"v", c.value}};
 }
 void from_json(nlohmann::json const& j, ColumnData& c) {
-  SafeGetTo(c.value, j, "v");
+  SafeGetToWithNullable(c.value, c.is_null, j, "v");
```

Comment:
If `c.value` was `absl::optional<std::string>` you could just use overloading on `SafeGetTo`?